wp-content/themes/theme_name/inc/media.php

```diff
@@ -14,6 +14,15 @@
    */
   function cc_mime_types( $mimes ) {
     $mimes['svg'] = 'image/svg+xml';
+    // unset($mimes['doc']);
+    // unset($mimes['docx']);
+    // unset($mimes['xls']);
+    // unset($mimes['xlsx']);
+    // unset($mimes['psd']);
+    // unset($mimes['ppt']);
+    // unset($mimes['pptx']);
+    // unset($mimes['pps']);
+    // unset($mimes['ppsx']);
     return $mimes;
```

Comment:
We could add a filter here so that we can hook into this function in plugins if necessary. Something like `return apply_filters( 'additional_mime_types', $mimes );`. This way we can use `additional_mime_types` hook later on without actually changing code within the theme. 

Also we should check if there is a filter for this in the core already.